src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -0,0 +1,116 @@
+package frc.robot.constants;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+
+//IDs and stuff
+public final class RobotConstants {
+    public static final class NeoMotorConstants {
+        public static final double kFreeSpeedRpm = 5676;
+    }
+    public static final class DriveConstants {
+        //placeholder CAN IDs, fix these later
+
```

Comment:
Everything under this line should not be in develop until it gets merged in by drivetrain. Some of them like the PID values should be in RobotConfig